test-common/src/main/java/org/apache/kafka/common/test/KafkaClusterTestKit.java

```diff
@@ -602,6 +637,13 @@ public void close() throws Exception {
             waitForAllFutures(futureEntries);
             futureEntries.clear();
             Utils.delete(baseDirectory);
+            jaasFile.ifPresent(f -> {
+                try {
```

Comment:
To enhance readability, we should explore alternatives to nested try-catch blocks, even if it involves using a more traditional coding style. for example: `if (jaasFile.isPresent()) Utils.delete(jaasFile.get());`